docker-storage-setup.1

```diff
@@ -83,6 +83,13 @@ DEVICE_WAIT_TIMEOUT:
            device. Default is 60 seconds. One can disable the wait by
            specifying a value of 0.
 
+OVERRIDE_SIGNATURES:
```

Comment:
What about "OVERWRITE_SIGNATURES" or "WIPE_SIGNATURES"?  "Override" implies to me that the signature would ordinarily be used for something, unless this option is set.